layers/README.md

```diff
@@ -119,3 +129,52 @@ lo4 = LayerOps(score=lambda x: '; '.join(x),
 out_layer6 = lo4.process([demo2, demo3])                      # Trigger processing on a list of demo2 and demo0
 out_layer6.to_file("C:\demo_layer6.json")                     # Save combined comment layer to file
 ```
+
+## to_excel.py
+to_excel.py provides the ToExcel class, which is a way to export an existing layer file as an Excel 
+spreadsheet. The ToExcel class has an optional parameter for the initialization function, that 
+tells the exporter to build the output matrix based on live data from cti-taxii.mitre.org. It 
+can also use cached subtechniques data. If neither of these is used, it will default to pulling from the 
```

Comment:
I recommend revising "cached subtechniques data". Sub-techniques are being released very soon and as per my prior comment cached isn't really the right term. Probably "local data" is sufficient. Also, sub-techniques should always be hyphenated.